modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -0,0 +1,86 @@
+:_content-type: PROCEDURE
+:description: Starting a new workspace with a clone of a Git repository
+:keywords: start-new-workspace, start-a-new-workspace, how-to-start-new-workspace, how-to-start-a-new-workspace, starting-a-new-workspace, clone-git-repository, clone-a-git-repository, how-to-start-workspace, how-to-start-a-workspace
+:navtitle: Starting a new workspace with a clone of a Git repository
+// :page-aliases:
+
+[id="starting-a-new-workspace-with-a-clone-of-a-git-repository_{context}"]
+= Starting a new workspace with a clone of a Git repository
+
+Working with {prod-short} in your browser involves multiple URLs:
+
+* URL of your organization's {prod-short} instance (as part of all the following URLs)
+* URL of the *Workspaces* page of your {prod-short} dashboard (workspace control panel)
+* [.underline]#URLs for starting a new workspace#
+* URLs of your workspaces in use
+
+With {prod-short}, you can visit a URL in your browser to start a new workspace that contains a clone of a Git repository. So you can have your remote origin or fork cloned into a new starting workspace. Or you can have a common repository of a project cloned into a new starting workspace. With such Git providers as GitHub and GitLab, you can also specify the branch for the new starting workspace to checkout.
+
+NOTE: The *Create Workspace* page of your {prod-short} dashboard has the *Git Repo URL ** field as the limited-functionality GUI alternative for entering the URL of a Git repository to start a new workspace.
```

Comment:
```suggestion
NOTE: The *Create Workspace* page of your {prod-short} dashboard has the *Git Repo URL* field as the limited-functionality GUI alternative for entering the URL of a Git repository to start a new workspace.
```

Reply:
:) This is the actual UI label: **Git Repo URL** *.